### PR TITLE
Allow avatar positioning from QML

### PR DESCRIPTION
**Pull request by jherico**

### CODE REVIEW

**Comment by jherico** [comment]
https://app.asana.com/0/26225263936266/92906674159839

**Comment by hifi-gustavo** [comment]
Build console output for commit: b4dd235c386c3c877ca787de841d2219e362dc46 available at: https://gist.github.com/4b095d5ea0cfc70b21e3<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7171.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7171.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 2258a9cad95a553ef7b4d162bbbefaffd456d22c available at: https://gist.github.com/4b095d5ea0cfc70b21e3<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7171.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7171.dmg

File: interface/src/avatar/MyAvatar.cpp [hunk 1]
```diff
@@ -123,8 +123,11 @@ MyAvatar::MyAvatar(RigPointer rig) :
     }
 
     // connect to AddressManager signal for location jumps
-    connect(DependencyManager::get<AddressManager>().data(), &AddressManager::locationChangeRequired,
-            this, &MyAvatar::goToLocation);
+    connect(DependencyManager::get<AddressManager>().data(), &AddressManager::locationChangeRequired,
```
Comment (zzmp) [inline interface/src/avatar/MyAvatar.cpp]: Is this necessary because you've overloaded goToLocation?

Comment (jherico) [inline interface/src/avatar/MyAvatar.cpp]: yes. Using the old code it couldn't distinguish between the two overloads of goToLocation.

**Comment by OmegaHeron** [comment]
https://alphas.highfidelity.io/t/ui-using-qml-in-high-fidelity/10195/20 Indicates this works proper.

**Pull request merged**